apps/docs/src/app/core/component-docs/link/examples/link-example.component.html

```diff
@@ -1,18 +1,21 @@
-<a [routerLink]="['./']" fd-link>Standard Link</a> <br /><br />
-<a [routerLink]="['./']" fd-link [emphasized]="true">Emphasized Link</a>
+<a [routerLink]="['./']" fd-link aria-label="Standard">Standard Link</a> <br /><br />
+<a [routerLink]="['./']" fd-link [emphasized]="true" aria-label="Emphasized">Emphasized Link</a>
 <br /><br />
 <a fd-link [disabled]="true" aria-disabled="true">Disabled Link</a>
 <br /><br />
-<a [routerLink]="['./']" fd-link>
+<a [routerLink]="['./']" fd-link aria-label="Icon right">
     Icon Right Link
     <fd-icon [glyph]="arrowRight$ | async"></fd-icon>
 </a>
 <br /><br />
-<a [routerLink]="['./']" fd-link>
+<a [routerLink]="['./']" fd-link aria-label="Icon left">
     <fd-icon [glyph]="arrowLeft$ | async"></fd-icon>
     Icon Left Link
 </a>
 <br /><br />
-<div style="background-color:#314a5e;padding:10px">
-    <a fd-link [routerLink]="['./']" fragment="inverted" [inverted]="true">Inverted Link</a>
-</div>
\ No newline at end of file
+
+<fd-shellbar>
```

Comment:
In my opinion it's an overkill to bring the Shellbar component in the documentation. I'd suggest using exactly the same approach as in fund-styles. Also, you should not be using the inverted link inside the Shellbar logo component.